src/onedrive.d

```diff
@@ -98,7 +98,13 @@ final class OneDriveApi
 		if (cfg.getValueBool("debug_https")) {
 			http.verbose = true;
 			.debugResponse = true;
-        }
+		}
+
+		// Custom User Agent
+		if (cfg.getValueString("user_agent") != "") {
+			http.setUserAgent = cfg.getValueString("user_agent");
+		}
```

Comment:
```
// Custom User Agent
		if (cfg.getValueString("user_agent") != "") {
			http.setUserAgent = cfg.getValueString("user_agent");
		} else {
			http.setUserAgent = "OneDrive Client for Linux " ~ strip(import("version"));
		}
```

Reply:
an option or no ?

Reply:
Yes, strongly agree to do this! It also shows a bit more presence to the servers ;-)